citi-onboarding/celeiro-dos-bambas
# Feature/deleting server

```
## What I did

- I've  deleted the server folder 
- I've moved the client archives out of the client folder, and deleted the client folder 

## How to test

Copy the link in the github from "celeiro-de-bambas-server" 
Go to the terminal 
Create a folder for the server project
Go to the folder
Git clone <link> 

Then 

In the folder of the project 

1.1. yarn install
1.2. yarn develop
1.3. Check if you are loged

In the folder "celeiro-dos-bambas"

2.1 yarn install
2.2 yarn yarn start

Check if the components looks just like the mockup in different screen sizes

Read the changed files section on GitHub

Test the things that you think are worth testing, even the ones that are not in this description
